app/repositories/application/GeneralApplicationRepoBSONReader.scala

```diff
@@ -97,11 +97,11 @@ trait GeneralApplicationRepoBSONReader extends CommonBSONDocuments {
       val overallScore = ca.flatMap(_.getAs[Double]("overallScore"))
 
       val se = pe.flatMap(_.getAs[BSONDocument]("schemes-evaluation"))
-      val commercial = se.flatMap(_.getAs[String](Schemes.Commercial).map(Result(_).toPassmark))
-      val digitalAndTechnology = se.flatMap(_.getAs[String](Schemes.DigitalAndTechnology).map(Result(_).toPassmark))
-      val business = se.flatMap(_.getAs[String](Schemes.Business).map(Result(_).toPassmark))
-      val projectDelivery = se.flatMap(_.getAs[String](Schemes.ProjectDelivery).map(Result(_).toPassmark))
-      val finance = se.flatMap(_.getAs[String](Schemes.Finance).map(Result(_).toPassmark))
+      val commercial = se.flatMap(_.getAs[String](SchemeId("Commercial").value).map(Result(_).toPassmark))
```

Comment:
Should these be in config somewhere?  Or maybe just defined once in microserviceConfig?